api/python/provisioner/utils.py

```diff
@@ -218,3 +222,26 @@ def repo_tgz(
     run_subprocess_cmd(cmd)
 
     return dest
+
+
+# Validates the hostname string in config.ini
+# against hostname in CLI args
+def node_hostname_validator(
+    nodes: List,
+    config_path: str
+):
+    logger.debug(f"Validating list of nodes: {'srvnode-1 : '.join(nodes)}")
+    logger.debug(f"Config file path: {config_path}")
+
+    parser_obj = configparser.ConfigParser()
+    parser_obj.read(config_path)
+
+    for section in parser_obj.sections():
+        if "srvnode" in section:
+            node_index = int(parser_obj[section].split("-")) - 1
```

Comment:
split command will result in list so we need only numder from srvnode-1 so 

 `node_index = int(parser_obj[section].split("-")) - 1`
 this line should be something like:
`node_index = int(section.split("-")[1]) - 1`